.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
We usually do not commit IDE config files.
Adding `.vscode` to `.gitignore` might be a good idea though.